src/dmd/declaration.d

```diff
@@ -1820,17 +1822,31 @@ extern (C++) class BitFieldDeclaration : VarDeclaration
                   ad.alignsize < memalignsize)
                 ad.alignsize = memalignsize;
         }
-
-        if (fieldWidth == 0)
+        else
         {
-            fieldState.inFlight = false;
-            return;
+            if (ad.alignsize == 0)
+                ad.alignsize = 1;
+            if (fieldWidth == 0)
+            {
+                if (fieldState.inFlight && !isunion)
+                {
+                    // documentation says align to next int
```

Comment:
Link to documentation?

Reply:
See BitFieldStyle in target.d, which has the links

Reply:
https://docs.microsoft.com/en-us/cpp/c-language/c-bit-fields?view=msvc-160

Reply:
Thanks Walter